### PR TITLE
Feature: Mapping project entities and relationships

### PR DESCRIPTION
**Pull request by eapg**
* Mapping project entities
* creating relationships between entities

### CODE REVIEW

File: src/tests/core/engine/app_engine_processor_integration_test.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
 from unittest import TestCase, mock
 
 from src.core.engine.app_engine_processor import AppEngineProcessor
+from src.lib.entities.sqlalchemy_orm_mapping import UserEntity
 
 
 class AppEngineProcessorIntegrationTest(TestCase):
     def test_process_run(self):
+
```
Comment (reynoldmorel) [inline src/tests/core/engine/app_engine_processor_integration_test.py:9]: Remove this change

File: src/tests/core/engine/app_engine_processor_integration_test.py [hunk 2]
```diff
@@ -1,10 +1,12 @@
 from unittest import TestCase, mock
 
 from src.core.engine.app_engine_processor import AppEngineProcessor
+from src.lib.entities.sqlalchemy_orm_mapping import UserEntity
```
Comment (reynoldmorel) [inline src/tests/core/engine/app_engine_processor_integration_test.py:4]: Remove this change

**Review by reynoldmorel — CHANGES_REQUESTED** [review]

File: src/tests/core/engine/app_engine_processor_integration_test.py (hunk 1 above)
Comment (eapg) [reply src/tests/core/engine/app_engine_processor_integration_test.py:9]: done

File: src/tests/core/engine/app_engine_processor_integration_test.py (hunk 2 above)
Comment (eapg) [reply src/tests/core/engine/app_engine_processor_integration_test.py:4]: done

File: src/lib/entities/sqlalchemy_orm_mapping.py [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+
+# Constraints
+
+
+def add_created_and_updated_relationships(cls):
+    cls.created = relationship(User, foreign_keys=[cls.created_by])
+    cls.updated = relationship(User, foreign_keys=[cls.updated_by])
+
+
+add_created_and_updated_relationships(User)
```
Comment (reynoldmorel) [inline src/lib/entities/sqlalchemy_orm_mapping.py:247]: I don't see `updated` and `created` for `AbstractEntity`

**Review by reynoldmorel — CHANGES_REQUESTED** [review]

**Review by reynoldmorel — APPROVED** [review]

**Pull request merged**